src/pymovements/events/properties.py

```diff
@@ -94,15 +94,15 @@ def dispersion(
 
     Parameters
     ----------
-    position_column
+    position_column: str
         The column name of the position tuples.
-    n_components:
+    n_components: int
         Number of positional components. Usually these are the two components yaw and pitch.
 
-    Raises
```

Comment:
We shouldn't remove this section

Reply:
pydoclint complains that the function does not have a raise in it, it just stupidly checks for raises within the scope of function :-( -- ignore the pydoclint error?